packages/whats-new/types.d.ts

```diff
@@ -19,6 +24,9 @@ declare module '@wordpress/rich-text' {
 		start?: number | undefined;
 		text: string;
 	}
+
+	// Fixed node_modules/@wordpress/components/build-types/autocomplete/types.d.ts(6,15): error TS2305: Module '"@wordpress/rich-text"' has no exported member 'RichTextValue'.
+	export type RichTextValue = Record< string, unknown >;
```

Comment:
I found another solution -- we can actually remove this entire `declare module '@wordpress/rich-text'` block, because the types are now included by the package! (This override seems to be causing the issue.)

Reply:
Yes I also noticed that but I'm not sure how to fix the issue as the type seems not to come from the package 🤔

Reply:
What confused me originally is that `RichTextValue` is provided by current versions of the package, but I think the overrides we had were breaking that. We needed the overrides previously because the types were incorrect and coming from DefinitelyTyped, but now `@wordpress/components` bundles types, so it's all included.

Reply:
But it seems to work in #78711 after I removed this module declaration